zingoconfig/src/lib.rs

```diff
@@ -116,12 +116,38 @@ pub struct ZingoConfig {
     pub logfile_name: PathBuf,
 }
 impl ZingoConfigBuilder {
-    pub fn set_wallet_dir(mut self, dir: PathBuf) -> Self {
-        self.wallet_dir = Some(dir);
+    /// the proxy server we download blockchain information from
+    /// # Examples
+    /// ```
+    /// use zingoconfig::ZingoConfigBuilder;
+    /// use http::Uri;
+    /// assert_eq!(ZingoConfigBuilder::default().set_lightwalletd_uri(("https://zcash.mysideoftheweb.com:19067").parse::<Uri>().unwrap()).lightwalletd_uri.clone().unwrap(), "https://zcash.mysideoftheweb.com:19067");
+    /// ```
+    pub fn set_lightwalletd_uri(&mut self, lightwalletd_uri: http::Uri) -> &Self {
+        self.lightwalletd_uri = Some(lightwalletd_uri);
         self
     }
-    pub fn set_lightwalletd(mut self, lightwalletd_uri: http::Uri) -> Self {
-        self.lightwalletd_uri = Some(lightwalletd_uri);
+    /// # Examples
+    /// ```
+    /// use zingoconfig::ZingoConfigBuilder;
+    /// use zingoconfig::ChainType::Testnet;
+    /// assert_eq!(ZingoConfigBuilder::default().set_chain(Testnet).create().chain, Testnet);
+    /// ```
+    pub fn set_chain(&mut self, chain: ChainType) -> &Self {
+        self.chain = chain;
+        self
+    }
+    /// the proxy server we download blockchain information from
```

Comment:
This is just copied from elsewhere and is incorrect.

Reply:
fixed